Podfile

```diff
@@ -9,5 +9,6 @@ target 'HighFlexDemo' do
 
   # Pods for HighFlexDemo
   # Use for local development
-  pod 'BerbixAdvanced', '~> 0.1.2'
+  # pod 'BerbixAdvanced', :path => '../ios-toolkit/BerbixAdvancedDev.podspec'
```

Comment:
should we remove this comment? `ios-toolkit` is an out of date name. I also noticed the README title includes `toolkit`, so would be worth correcting that while you're at it.